presto-pinot-toolkit/pom.xml

```diff
@@ -682,6 +686,9 @@
                                     <ignoredDependency>io.grpc:::</ignoredDependency>
                                     <ignoredDependency>commons-codec:commons-codec::</ignoredDependency>
                                 </ignoredDependencies>
+                                <ignoredUnusedDeclaredDependencies>
+                                    <ignoredUnusedDeclaredDependency> org.apache.lucene:lucene-core:jar:8.10.0</ignoredUnusedDeclaredDependency>
+                                </ignoredUnusedDeclaredDependencies>
```

Comment:
I found this change didn't seem to be necessary to build `presto-pinot-toolkit` without errors.

```
 ./mvnw clean install -pl presto-pinot-toolkit -am -DskipUI -DskipTests
```